documentation/src/main/asciidoc/cheat-sheets/cache-configuration.asciidoc

```diff
@@ -117,17 +117,17 @@ Eviction occurs after the container size exceeds the maximum count.
 </distributed-cache>
 ----
 
-.Eviction strategies
-* `REMOVE` {brandname} performs eviction. This is the default strategy when you define a size for the data container. Because not all nodes evict the same entries you should use eviction with persistent storage to avoid data inconsistency.
+===== Eviction strategies
```

Comment:
```suggestion
==== Eviction strategies
```

Reply:
Eviction strategies are a subsection of Eviction. I think we should keep it as it is 
`==== Eviction`
`===== Eviction strategies`